libimage/manifests/manifests_test.go

```diff
@@ -335,6 +338,30 @@ func TestPushManifest(t *testing.T) {
 	_, _, err = list.Push(ctx, destRef, options)
 	assert.Nilf(t, err, "list.Push(four specified)")
 
+	bogusDestRef, err := alltransports.ParseImageName("docker://localhost/bogus/dest:latest")
+	assert.Nilf(t, err, "ParseImageName()")
```

Comment:
Probably want to use `assert.NoErrorf()` here, since we know that `err` is an error if it isn't `nil`.